### ISSUE TITLE
Blank Data

### ISSUE THREAD

**Issue by QISPSK** [EyeSeeTea/SurveillanceCambodiaApp#91]
We've started rolling out the application to providers however it appears that provider Cheng Long (02-00-61) has sent over 2000 blank surveys yesterday. We have yet to call the provider to confirm what is happening, but it appears the blank data issue has not yet been solved.

Thankfully, he seems to be the only provider with this issue. Will update as soon as we can call the provider today.

**Comment by ifoche** [comment]
@QISPSK I've also seen that problem in QIS_test_int OU, so we give it the maximum priority to try to find out which situation is generating those blanks.

**Comment by ifoche** [comment]
@QISPSK can you confirm that device in Cheng Long (02-00-61) is using Google Play app in its last version?

**Comment by QISPSK** [comment]
He should be. Technically he has no way to check as we've locked down settings on the phone for providers. Just in case, I've sent a request from Google Play to remotely install the latest version just now so hopefully that fixes the issue.

Panharith was able to contact the provider and had the following to say:

"On the excel that I have extracted from DHIS2 shown me that one negative case came up with blank location and time stamp for that blank location is the same to all 2902 blank data (2016-07-10 07:37:27.407). I have contacted to him this moment and he also mentioned that internet service at his outlet is working well and no application crash anyway."

Perhaps the issue is linked somehow to GPS location?
